cpp/comms/std/src/cuML_std_comms_impl.cpp

```diff
@@ -39,6 +39,9 @@ constexpr bool UCX_ENABLED = false;
 #include <cuML_comms.hpp>
 #include <exception>
 #include <memory>
+
```

Comment:
There are several places in the comms and in `ucp_helper.h` that have couts and I think they should be replaced by either INFO or DEBUG logs (depending on whichever ACTIVE LEVEL is agreed upon before this is merged).

Reply:
Makes sense. Done.